### DISCUSSION TITLE
Got error about sh_shouvo/materials.git when running yarn install

### DISCUSSION THREAD

**Discussion by thanhpd56** [CrewHRM/CrewHRM#96] [General]
Hello,
I got this error after run npm install or yarn install.
"Arguments: ls-remote --tags --heads ssh://git@bitbucket.org/sh_shouvo/materials.git
The requested repository either does not exist or you do not have access
"
Please help.

**Comment by sekanderb** [comment]
Hello @thanhpd56,
This is one of our private libraries. It is not possible to make it public.

**Reply by thanhpd56** [reply]
Hi @sekanderb
Thanks for your reply.
So how can I folk or contribute to this project?

**Reply by thanhpd56** [reply]
Even I cloned this project, I still can not run it.
